### PR TITLE
SE-1263: add prorated across SHK

### PR DESCRIPTION
**Pull request by slemasne**
(1) Adding example of Prorated Transaction Type

### CODE REVIEW

File: examples/features/core-lusid/Sub-holding Keys.ipynb [hunk 1]
```diff
@@ -2,8 +2,24 @@
  "cells": [
```
Comment (thomasdotred) [inline examples/features/core-lusid/Sub-holding Keys.ipynb:2]: <p><code>FBN_SECRETS_PATH_CALCULATION</code>  not <code>FBN_SECRETS_PATH</code> , why the change?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (inwaves) [inline examples/features/core-lusid/Sub-holding Keys.ipynb:2]: <p>This should be the same heading level as "1. Sub-Holding Keys" (I think it's h3 instead of h1)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (inwaves) [inline examples/features/core-lusid/Sub-holding Keys.ipynb:2]: <p>Should this point to the standard environment variable for secrets path? (wondering if you changed this just for testing purposes)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (inwaves) [inline examples/features/core-lusid/Sub-holding Keys.ipynb:2]: <p>As above the heading level should match points 1., 2., 3.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (inwaves) [inline examples/features/core-lusid/Sub-holding Keys.ipynb:2]: <p>Small typo on allocation, could change if this is client-facing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (thomasdotred) [inline examples/features/core-lusid/Sub-holding Keys.ipynb:2]: <p>In TESCO we initially have:</p><p>ftse_tracker = 6000</p><p>food_retail = 4000</p><p> </p><p>(2:1)</p><p> </p><p>the we book 6000 units of Tesco</p><p> </p><p>how does <code>TransactionConfiguration/default/AllocationMethod</code> know to allocate to the below breakdown, does it just take the original ratio across strategies? </p><p>ftse_tracker = 4000</p><p>food_retail = 2000</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (slemasne) [reply examples/features/core-lusid/Sub-holding Keys.ipynb:2]: Good catch, typo.

Comment (slemasne) [reply examples/features/core-lusid/Sub-holding Keys.ipynb:2]: <p>agreed</p>

Comment (slemasne) [reply examples/features/core-lusid/Sub-holding Keys.ipynb:2]: <p>Good catch, typo</p>

Comment (slemasne) [reply examples/features/core-lusid/Sub-holding Keys.ipynb:2]: <p>agreed</p>

Comment (slemasne) [reply examples/features/core-lusid/Sub-holding Keys.ipynb:2]: <p>thanks, corrected</p>

Comment (slemasne) [reply examples/features/core-lusid/Sub-holding Keys.ipynb:2]: <p>Thats right. </p><p>It takes the ratio. </p>

**Review by inwaves — APPROVED** [review]
